vault/changelog.release.2021-11-23.md

```diff
@@ -0,0 +1,69 @@
+---
+id: Ku1qjbDYlsxk4rfMoFEza
+title: "0.70"
+desc: ''
+updated: 1637691256859
+created: 1637687416614
+---
+
+Dendron 0.70 has sprouted  🌱
+
+Dendronites can now use `Dendron: Initialize Workspace` to create [[native workspaces|dendron.topic.workspace.native]], variable substitution is possible in the desc note frontmatter, and improvements have been made to task note decorators (the task info that magically renders whenever a note link is pointing to a task note). Jump in and take a look!
+
+This release also includes a variety of fixes and enhancements to publishing, among other improvements.
+
+### Highlights
+- feat(workspace): Initialize Workspace command can create native workspaces ([[docs|dendron.topic.workspace.native]])
+- enhance(publish): Better formatting of tables on published site
+- fix(publish): syntax highlighting for code blocks
+- fix(publish): make mermaid work consistently on published sites
+- fix(publish): enable katex on published site 
+
+### Everything Else
+- enhance(markdown): expose `desc` frontmatter property for variable substitution ([[docs|dendron.topic.markdown#frontmatter-variable-substitution]])
+  - View the [difference](https://github.com/dendronhq/dendron-site/pull/284/files)
```

Comment:
what about just going back to (diff) here? keeps the release notes concise

Reply:
Good call. I'm reflecting that in both changelog and release notes now.